frankDoc/src/main/java/nl/nn/adapterframework/frankdoc/model/AttributeExcludedSetter.java

```diff
@@ -0,0 +1,77 @@
+/* 
+Copyright 2021 WeAreFrank! 
+
+Licensed under the Apache License, Version 2.0 (the "License"); 
+you may not use this file except in compliance with the License. 
+You may obtain a copy of the License at 
+
+    http://www.apache.org/licenses/LICENSE-2.0 
+
+Unless required by applicable law or agreed to in writing, software 
+distributed under the License is distributed on an "AS IS" BASIS, 
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. 
+See the License for the specific language governing permissions and 
+limitations under the License. 
+*/
+
+package nl.nn.adapterframework.frankdoc.model;
+
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.stream.Collectors;
+
+import org.apache.commons.lang3.StringUtils;
+import org.apache.logging.log4j.Logger;
+
+import nl.nn.adapterframework.frankdoc.doclet.FrankClass;
+import nl.nn.adapterframework.frankdoc.doclet.FrankMethod;
+import nl.nn.adapterframework.util.LogUtil;
+
+class AttributeExcludedSetter {
+	private static Logger log = LogUtil.getLogger(AttributeExcludedSetter.class);
+
+	private Set<String> namesExcludedDueToIgnoredInterface = new HashSet<>();
+
+	AttributeExcludedSetter(FrankClass clazz) {
+		String ignoredInterface = clazz.getJavaDocTag(FrankElement.JAVADOC_IGNORE_TYPE_MEMBERSHIP);
+		if(ignoredInterface != null) {
+			if(StringUtils.isBlank(ignoredInterface)) {
+				log.warn("Javadoc tag {} requires an argument that refers a Java interface", FrankElement.JAVADOC_IGNORE_TYPE_MEMBERSHIP);
+			} else {
+				log.trace("Have Javadoc tag {} that refers to interface [{}]", () -> FrankElement.JAVADOC_IGNORE_TYPE_MEMBERSHIP, () -> ignoredInterface);
+				AttributesFromInterfaceRejector rejector = new AttributesFromInterfaceRejector(new HashSet<>(Arrays.asList(ignoredInterface)));
```

Comment:
Misschien dat het mooier is om een extra constructor toe te voegen voor wanneer je maar 1 waarde hebt. Ik zie overigens niet waar de AttributesFromInterfaceRejector met meerdere waardes gebruikt wordt?

Reply:
Gedaan. Er is inderdaad altijd maar één waarde bij het construeren van een AttributesFromInterfaceRejector. Ik heb de constructor die een Set vraagt weggehaald.